### PR TITLE
Profilebanner storybook

### PR DESCRIPTION
**Pull request by IITvamp**
## Description
passing value via props to the profile banner components. No changes in the UI in this pr, just changed hardcoded data to props.

## Related Issue
#294 

## Screenshots or GIF (In case of UI changes):
image
image
image
image

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/components/ProfileBanner/Organization/index.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ import dp from "../../../assets/images/demoperson1.jpeg";
 import BannerDemo from "../../../assets/images/OrgBannerDemo.png";
 import iconbuttonImage from "../../../assets/images/unfilled3holes.svg";
 
-export default function Banner() {
+export default function Banner({ bannerImage, profileImage, name, story, followers, contributors, feed }) {
```
Comment (Niweera) [inline src/components/ProfileBanner/Organization/index.js:11]: did you use prettier in this line?

**Comment by charithccmc** [comment]
@IITvamp Pls add a gif of the Profilebanner

File: src/components/ProfileBanner/Organization/index.js (hunk 1 above)
Comment (IITvamp) [reply src/components/ProfileBanner/Organization/index.js]: I didn't. I have pushed changes after running prettier on all the files related to the pr.

Comment (Niweera) [reply src/components/ProfileBanner/Organization/index.js]: can you double check, the line is long and it's on a single line which should be broken down to new lines.

Comment (IITvamp) [reply src/components/ProfileBanner/Organization/index.js]: I have made a new commit after running prettier. https://github.com/scorelab/Codelabz/commit/43d6a1e6a901cb728aa65b0923f189230941b2b9
Please check. This line has been broken to multiple short lines.

**Review by Niweera — APPROVED** [review]

**Review by Niweera — APPROVED** [review]

**Pull request merged**